### ISSUE TITLE
Accessibility support

### ISSUE THREAD

**Issue by lonely-boy0** [ustbhuangyi/better-scroll#1334]
Does better-scroll have good accessibility support? I am using iOS accessibility features, but I cannot vertically scroll using the three-finger gesture or the continuous single tap gesture.
